ui/tests/integration/components/search-select-test.js

```diff
@@ -329,4 +337,100 @@ module('Integration | Component | search select', function (hooks) {
       'onClick is called with array of objects with isNew true on new item'
     );
   });
+
+  test('it returns custom object if passObject=true and multiple objectKeys', async function (assert) {
```

Comment:
Great coverage of new functionality! I'd like to see one more test where if we use a model without extra keys and pass in `objectKeys` it fails gracefully, and/or if it has extra keys but `objectKeys` includes one that isn't present/is falsy

Reply:
Great suggestion! Will do